### PR TITLE
Feature/story test/develop

### PR DESCRIPTION
**Pull request by dumebi**
merged `poly-wanna-diamond` with `story-test/master` and fixed all conflicts

### CODE REVIEW

File: deployments/mainnet/.latestDeploymentBlock [hunk 1]
```diff
@@ -1 +1 @@
-12647881
\ No newline at end of file
+12692220
```
Comment (sypeer) [inline deployments/mainnet/.latestDeploymentBlock:1]: mainnet?

File: hardhat.config.ts [hunk 2]
```diff
@@ -176,16 +176,16 @@ export default <HardhatUserConfig>{
...
+    // polygon: networkConfig({
+    //   url: networkUrls.polygon,
+    //   chainId: 137,
+    //   live: true,
+    // }),
+    // polygon_mumbai: networkConfig({
+    //   url: networkUrls.polygon_mumbai,
+    //   chainId: 80001,
+    //   live: true,
+    // }),
```
Comment (sypeer) [inline hardhat.config.ts:188]: why is this commented out?

File: test/helpers/chai-helpers.ts [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
+//       if (ethers.BigNumber.isBigNumber(obj)) {
+//         const expected = ethers.BigNumber.isBigNumber(args[0])
+//           ? args[0].toString()
+//           : args[0]
+//         new chai.Assertion(obj.toString()).to.eql(expected, ...args.slice(1))
+//       } else {
+//         _super.apply(this, args)
+//       }
+//     }
+// )
```
Comment (sypeer) [inline test/helpers/chai-helpers.ts:18]: Please delete if the file is not going to be used going forward

**Review by sypeer — COMMENTED** [review]
Looking good so far!

File: deployments/mainnet/.latestDeploymentBlock (hunk 1 above)
Comment (dumebi) [reply deployments/mainnet/.latestDeploymentBlock:1]: i think so, will merge the latest from develop

File: hardhat.config.ts (hunk 2 above)
Comment (dumebi) [reply hardhat.config.ts:188]: my mistake

File: test/helpers/chai-helpers.ts (hunk 3 above)
Comment (dumebi) [reply test/helpers/chai-helpers.ts]: done

**Review by mkhoury93 — CHANGES_REQUESTED** [review]
do we need `liquidate.tests.ts` if it's all commented out?

**Pull request closed without merging**